packages/mdc-textfield/icon/README.md

```diff
@@ -29,13 +29,11 @@ Icons describe the type of input a text field requires. They can also be interac
 
 ### Usage within `mdc-text-field`
 
-Leading and trailing icons can be added to `mdc-text-field` as visual indicators
-as well as interaction targets. To do so, add the relevant classes
-(`mdc-text-field--with-leading-icon` or `mdc-text-field--with-trailing-icon`) to the root element, add
-an `i` element with your preferred icon, and give it a class of `mdc-text-field__icon`.
+Leading and trailing icons can be applied to text fields styled as `mdc-text-field--box` or `mdc-text-field--outlined`. To add an icon, add the relevant class (either `mdc-text-field--with-leading-icon` or `mdc-text-field--with-trailing-icon`) to the root element, add an `i` element with your preferred icon, and give it a class of `mdc-text-field__icon`.
 
-#### Leading:
+#### Leading icon
 
+In box text field:
```

Comment:
Pretty sure we've typically referred to this in the past as text field box? (e.g. in the demo page since it was first added)